docs.html

```diff
@@ -1591,43 +1591,105 @@ <h2 class="nav-logo">BinaryUI</h2>
-                        style="width: 100%; height: 267px; border:0; transform: scale(1); overflow:hidden;"
-                        sandbox="allow-scripts allow-same-origin">
-                    </iframe>
+                    <div class="snippets-display">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%2874%2C144%2C226%2C1%29&t=material&wt=none&l=auto&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=0px&ph=0px&ln=false&fl=1&fm=Fira+Code&fs=14px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522navigation%2520sm-rounded%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%2520%2520%2520%253Cdiv%2520class%253D%2522nav-left%2520logo%2522%253E%250A%2520%2520%2520%2520%2520%2509%253Ch2%2520class%253D%2522nav-logo%2522%253EBinaryUI%253C%252Fh2%253E%250A%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522nav-center%2520searchbar%2522%253E%250A%2520%2520%2520%2520%2509%253Cinput%2520type%253D%2522search%2522%2520class%253D%2522input%2520input-search%2522%2520placeholder%253D%2522Search%2522%252F%253E%250A%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%250A%2520%2520%253Cdiv%2520class%253D%2522nav-right%2522%253E%250A%2520%2520%2520%2520%2520%2520%253Cbutton%2520class%253D%2522btn%2520btn-primary%2520login-button%2520text-md%2522%253ELogin%253C%252Fbutton%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-container%2520icon-badge%2520%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-shopping-cart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-icon-number%2520%2520badge-status%2522%253E%25204%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%250A%2520%2520%2520%2520%2520%2520%253Cdiv%2520class%253D%2522badge-container%2520icon-badge%2522%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Ci%2520class%253D%2522fas%2520fa-heart%2522%253E%253C%252Fi%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%253Cspan%2520class%253D%2522badge-icon-number%2520%2520badge-status%2522%253E%25204%2520%253C%252Fspan%253E%250A%2520%2520%2520%2520%2520%2520%253C%252Fdiv%253E%250A%2520%2520%253C%252Fdiv%253E%250A%2520%2520%2520%2520%2520%2520%2520%2520%2520%2520%250A%253C%252Fdiv%253E"
```

Comment:
Can we use a shorter link for the iframe? We can login so that the link gets shorter. it gives a clean look

Reply:
Yeah, I know but I think lots of problems with carbon.